test/testHelper.ts

```diff
@@ -45,6 +45,12 @@ export default class ContextTestingHelper {
     return this._context.store[String(key)];
   }
 
+  createAsyncapiFile(fileName: string, template: string): void {
+    const filePath = path.resolve(process.cwd(), fileName);
+    const templatePath = path.resolve('assets/examples', template);
+    writeFileSync(filePath, readFileSync(templatePath, { encoding: 'utf-8' }), { encoding: 'utf-8' });
+  }
+
```

Comment:
You probably don't need this because `new` creates `asyncapi.yaml` file in the project directory by default so you can just use that file to run tests as it is added in the `.gitignore`. All you have to do is remove the file before the test is running.

Reply:
I have made the requested changes but haven't removed the prompt part yet.

Also, I have to implement a test for `start studio` do I need to open another PR for that or implement it in the same.
Kindly, provide me a little bit of information on the implementation of the `start studio` test.

Reply:
Open a different PR for `start studio` command obviously. We might now be able to test for `start studio` command. Have to look into it because start studio opens a web socket server and I have no Idea how you would test that.